cobalt/webdriver/testdata/simple_test.py

```diff
@@ -310,7 +310,7 @@ def ElementUniqueTest():
 
 
 def main():
-  MouseTest()
+  #MouseTest()
   ElementScreenShotTest()
   # ElementUniqueTest()
```

Comment:
Not related to this PR, but do you have any context why we comment this test out earlier?

Reply:
Oh, this script is just a helper script (showing how to do webdriver at the lowest level wire protocol) and this change was a result of me manually testing. I don't know, didn't look if we have this covered as part of the webdriver tests. I can revert this change.